uuidv8.go

```diff
@@ -258,3 +260,32 @@ func (u *UUIDv8) UnmarshalJSON(data []byte) error {
 	*u = *parsed
 	return nil
 }
+
+// Value implements the driver.Value interface for database writes.
```

Comment:
You can use this to allow link in the godoc

```suggestion
// Value implements the [driver.Valuer] interface for database writes.
```